src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -850,6 +855,11 @@ namespace ReportSizingManager {
 								AutosizeDes = 0.389 + 7684.0*RatedVolFlowPerRatedTotCap;
 							}
 						}
+
+						// check that the autosized SHR corresponds to a valid apperatus dew point (ADP) temperature
+						DesMassFlow = DataFlowUsedForSizing * PsyRhoAirFnPbTdbW( StdBaroPress, RatedInletAirTemp, RatedInletAirHumRat, CallingRoutine );
+						AutosizeDes = ValidateADP( CompType, CompName, RatedInletAirTemp, RatedInletAirHumRat, DataCapacityUsedForSizing, DesMassFlow, AutosizeDes, CallingRoutine );
+
```

Comment:
Right after SHR is autosized, silently test bypass factor to iterate on SHR if needed. Revised SHR is used instead and reported after this point.